server/src/test/java/io/crate/types/TypeCompatibilityTest.java

```diff
@@ -39,4 +39,5 @@ void test_get_common_type_for_text_with_len_limit_return_text_with_highest_lengt
         var commonType = TypeCompatibility.getCommonType(StringType.of(2), StringType.of(1));
         assertThat(commonType, is(StringType.of(2)));
     }
+
```

Comment:
apologies for the nitpick: since this file is not touched for the fix, it could be confusing when git checking the history (high level).